<?xml version="1.0" ?>
<testsuite errors="1" failures="0" name="TestDescuentos-20190826093000" tests="8" time="0.001">
	<testcase classname="TestDescuentos" name="test_dependientes_menor_cero" time="0.000"/>
	<testcase classname="TestDescuentos" name="test_descuento_familias" time="0.000"/>
	<testcase classname="TestDescuentos" name="test_descuento_menor_edad" time="0.000"/>
	<testcase classname="TestDescuentos" name="test_nombre_casos_prueba" time="0.000"/>
	<testcase classname="TestDescuentos" name="test_todos_valores_no_validos" time="0.000"/>
	<testcase classname="TestDescuentos" name="test_todos_valores_validos" time="0.000"/>
	<testcase classname="TestDescuentos" name="test_valor_cotizado_menor" time="0.000"/>
	<testcase classname="TestDescuentos" name="test_descuento_especial_no_aplica" time="0.001">
		<error message="Tuples differ: ('Descuento para clientes', 15.0) != ('No aplica', 0)

First differing element 0:
'Descuento para clientes'
'No aplica'

- ('Descuento para clientes', 15.0)
+ ('No aplica', 0)" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "test.py", line 44, in test_descuento_especial_no_aplica
    self.assertEqual(descuentos.calcular_descuento(70, 1500, 3), ('No aplica', 0))
AssertionError: Tuples differ: ('Descuento para clientes', 15.0) != ('No aplica', 0)

First differing element 0:
'Descuento para clientes'
'No aplica'

- ('Descuento para clientes', 15.0)
+ ('No aplica', 0)
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
